src/content/en/updates/2017/09/autoplay-policy-changes.md

```diff
@@ -132,6 +132,17 @@ get to the specific blog, so autoplay is allowed. However, the blog needs to
 explicitly delegate that privilege to the iframe in order for the content to
 autoplay.
 
+### Chrome enterprise policies
+
+It is possible to change this new autoplay behaviour with Chrome enterprise
+policies for use cases such as kiosks or unattended systems. Check out the
+[Configure policies and settings] help page to learn how to set these new
+autoplay related enterprise policies:
+
+- The ["AutoplayAllowed"] policy controls whether autoplay is allowed or not.
```

Comment:
Should these be backticked (`) instead of quoted?

Reply:
Policies are not `backticked` usually. See http://dev.chromium.org/administrators/policy-list-3#AutoplayAllowed and `chrome://policy`

Reply:
Is this invoked in code, or in a user interface? I suspect this should be both. `"AutoplayAllowed"`

Reply:
Then it's quotes, not backticks.